Removed data_model components in vignettes (Issue #170) (#180)
krlmlr/dm: vignettes/dm-visualization.Rmd

}
```

Once you have all your primary keys set and all foreign key relations defined, a graphical representation of your data model offers a condensed view of the tables and the relationships between the tables.

> The following functions can be used to visualize the `dm` object:^[The functions in this section rely heavily on related functions of [{datamodelr}](https://github.com/bergant/datamodelr).]

Reviewer: ```suggestion
The following functions can be used to visualize the `dm` object:^[The code for the functions in this section is borrowed from the [{datamodelr}](https://github.com/bergant/datamodelr) package.]
```
